### DISCUSSION TITLE
Add api docs plz.

### DISCUSSION THREAD

**Discussion by Abab-bk** [DaveGreen-Games/ShapeEngine#46] [Q&A]
Please add an api doc genereted by docfx, i add one in my project(used ShapeEngine), Work fine, but I want an online official version. it can help reading source code.

**Comment by SoloByte** [comment]
FYI, I added an [official documentation](https://davegreen-games.github.io/ShapeEngineDocs/v4.0.4/) a while back and I have just added a version selector for all Shape Engine releases + main branch.
